sopel/plugins/rules.py

```diff
@@ -475,7 +475,7 @@ def is_limited(
         time_limit: datetime.datetime,
     ) -> bool:
         """Determine if the rule hits the time limit."""
-        if not self.started_at and not self.ended_at:
+        if not self.started_at:
```

Comment:
Both of us missed this in #2297, huh?

Reply:
Technically, if `self.started_at` is `None`, then `self.ended_at` should be `None` too. However, this is an unnecessary check that made mypy grumpy (for good reason).

Reply:
my*grum*py 😉 got it